packages/backend-defaults/src/entrypoints/rootHttpRouter/rootHttpRouterServiceFactory.ts

```diff
@@ -130,7 +112,10 @@ const rootHttpRouterServiceFactoryWithOptions = (
         logger,
         lifecycle,
         healthRouter,
-        lifecycleMiddleware,
+        // TODO: delete this when the deprecated lifecycleMiddleware is removed
+        lifecycleMiddleware: (_req, _res, next) => {
+          next();
+        },
```

Comment:
This was added in the previous PR right? It's not been released in mainline yet so we can remove it straight away and update the existing changeset